<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      About &middot; Tiberiu Teșileanu
    
  </title>

  
  <link rel="canonical" href="https://ttesileanu.com/about">
  

  <!-- Fonts & Icons -->
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
  <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">

  <link rel="stylesheet" href="https://ttesileanu.com/public/css/poole.css">
  <link rel="stylesheet" href="https://ttesileanu.com/public/css/syntax.css">
  <link rel="stylesheet" href="https://ttesileanu.com/public/css/lanyon.css">
  <link rel="stylesheet" href="https://ttesileanu.com/public/css/from_alfolio.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">

  
    <link rel="shortcut icon" href="/public/img/favicon.png"/>
  

  
</head>


  <body>

    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior. -->
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">

<!-- Toggleable sidebar -->
<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>Neuroscience researcher with background in theoretical physics.
</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item" href="https://ttesileanu.com/">Home</a>

    

    
    
      
        
      
    
      
        
      
    
      
        
          <a class="sidebar-nav-item active" href="https://ttesileanu.com/about">About</a>
        
      
    
      
        
          <a class="sidebar-nav-item" href="https://ttesileanu.com/blog/index.html">Blog</a>
        
      
    
      
    
      
    
      
        
          <a class="sidebar-nav-item" href="https://ttesileanu.com/publications">Publications</a>
        
      
    
      
    
      
    
  </nav>

  <div class="sidebar-item">
    <p>
      © 2023 Tiberiu Teșileanu.
    </p>
  </div>
</div>


    <!-- Wrap is the content to shift when toggling the sidebar. We wrap the
         content to avoid any CSS collisions with our real content. -->
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="/" title="Home">Tiberiu Teșileanu</a>
            <small>a physicist working in neuroscience</small>
          </h3>
        </div>
      </div>

      <div class="container content">
        <div class="page">
  <h1 class="page-title">About</h1>
  <article>
  <div class="profile float-right">
<figure>
  <picture><source media="(max-width: 480px)" srcset="/public/img/prof_pic-480.webp"></source><source media="(max-width: 800px)" srcset="/public/img/prof_pic-800.webp"></source><source media="(max-width: 1400px)" srcset="/public/img/prof_pic-1400.webp"></source><!-- Fallback to the original file --><img class="img-fluid z-dept-1 rounded" src="/public/img/prof_pic.jpg" alt="prof_pic.jpg"></picture>
</figure>
    <div class="address">
      <p>Farley Building</p> <p>New York, NY 10001</p>

    </div>
  </div>

  <div class="clearfix">
    <p>I am a theoretical physicist turned neuroscientist, turning software engineer. Given my background, my approach to neuroscience has been based on overarching principles and precise, quantitative modeling—typically analytical results followed by numerical simulations.</p>

<p>My neuroscience work was focused on normative modeling of brain circuits, where some optimality criterion based on biologically relevant tasks was used to predict neural-circuit architectures, in particular those that use biologically plausible learning rules.</p>

<p>In what now seems like a different life, I worked in theoretical high-energy physics, focusing on various aspects of the AdS/CFT duality in string theory.</p>

<p>I am now starting a job as a Research Software Engineer in the CTRL team at <a href="https://www.meta.com" target="_blank" rel="noopener noreferrer">Meta</a>.</p>


  </div>

  
  <!-- Selected papers -->
          <div class="publications">
            <h2>selected publications</h2>
            <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-3 d-flex justify-content-center">
<!-- check for dark version of the image --><!-- automatically generate dark version name by adding '-dark' before ext --><div class="photo"><img class="photo-image" src="/public/img/pub/mypub_imgs_seq_seg.png"></div>
</div>

  <!-- Entry bib key -->
  <div id="Tesileanu2021" class="col-sm-9">
    <!-- Title -->
    <div class="title"><a href="http://arxiv.org/abs/2104.11852" target="_blank" rel="noopener noreferrer">Neural circuits for dynamics-based segmentation of time series</a></div>
    <!-- Author -->
    <div class="author">
<em>Tesileanu, T.</em>, Golkar, S., Nasiri, S., Sengupta, A.M., and Chklovskii, D.B.</div>

    <!-- Journal/Book title and date -->
    <div class="periodical">
      <em>arXiv</em> 2021
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
      <a href="https://arxiv.org/pdf/2104.11852.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
    </div>

    <!-- Hidden abstract block -->
    <div class="abstract hidden">
      <p>The brain must extract behaviorally relevant latent variables from the signals streamed by the sensory organs. Such latent variables are often encoded in the dynamics that generated the signal rather than in the specific realization of the waveform. Therefore, one problem faced by the brain is to segment time series based on underlying dynamics. We present two algorithms for performing this segmentation task that are biologically plausible, which we define as acting in a streaming setting and all learning rules being local. One algorithm is model based and can be derived from an optimization problem involving a mixture of autoregressive processes. This algorithm relies on feedback in the form of a prediction error and can also be used for forecasting future samples. In some brain regions, such as the retina, the feedback connections necessary to use the prediction error for learning are absent. For this case, we propose a second, model-free algorithm that uses a running estimate of the autocorrelation structure of the signal to perform the segmentation. We show that both algorithms do well when tasked with segmenting signals drawn from autoregressive models with piecewise-constant parameters. In particular, the segmentation accuracy is similar to that obtained from oracle-like methods in which the ground-truth parameters of the autoregressive models are known. We also test our methods on data sets generated by alternating snippets of voice recordings. We provide implementations of our algorithms at <a href="https://github.com/ttesileanu/bio-time-series" target="_blank" rel="noopener noreferrer">https://github.com/ttesileanu/bio-time-series</a>.</p>
    </div>
<!-- Hidden bibtex block -->
    <div class="bibtex hidden">
      <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Tesileanu2021</span><span class="p">,</span>
  <span class="na">image</span> <span class="p">=</span> <span class="s">{mypub_imgs_seq_seg.png}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">archiveprefix</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">arxivid</span> <span class="p">=</span> <span class="s">{2104.11852}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tesileanu, Tiberiu and Golkar, Siavash and Nasiri, Samaneh and Sengupta, Anirvan M. and Chklovskii, Dmitri B.}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1162/neco_a_01476}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{2104.11852}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--34}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Neural circuits for dynamics-based segmentation of time series}}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{biowta}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://arxiv.org/abs/2104.11852}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{https://arxiv.org/pdf/2104.11852.pdf}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span>
<span class="p">}</span></code></pre></figure>
    </div>
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-3 d-flex justify-content-center">
<!-- check for dark version of the image --><!-- automatically generate dark version name by adding '-dark' before ext --><div class="photo"><img class="photo-image" src="/public/img/pub/mypub_imgs_protein_sectors.png"></div>
</div>

  <!-- Entry bib key -->
  <div id="Tesileanu2015" class="col-sm-9">
    <!-- Title -->
    <div class="title"><a href="http://dx.plos.org/10.1371/journal.pcbi.1004091" target="_blank" rel="noopener noreferrer">Protein Sectors: Statistical Coupling Analysis versus Conservation</a></div>
    <!-- Author -->
    <div class="author">
<em>Tesileanu, T.</em>, Colwell, L.J., and Leibler, S.</div>

    <!-- Journal/Book title and date -->
    <div class="periodical">
      <em>PLOS Computational Biology</em> 2015
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
      <a href="https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1004091&amp;type=printable" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a>
    </div>

    <!-- Hidden abstract block -->
    <div class="abstract hidden">
      <p>Statistical coupling analysis (SCA) is a method for analyzing multiple sequence alignments that was used to identify groups of coevolving residues termed “sectors”. The method applies spectral analysis to a matrix obtained by combining correlation information with sequence conservation. It has been asserted that the protein sectors identified by SCA are functionally significant, with different sectors controlling different biochemical properties of the protein. Here we reconsider the available experimental data and note that it involves almost exclusively proteins with a single sector. We show that in this case sequence conservation is the dominating factor in SCA, and can alone be used to make statistically equivalent functional predictions. Therefore, we suggest shifting the experimental focus to proteins for which SCA identifies several sectors. Correlations in protein alignments, which have been shown to be informative in a number of independent studies, would then be less dominated by equence conservation.</p>
    </div>
<!-- Hidden bibtex block -->
    <div class="bibtex hidden">
      <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Tesileanu2015</span><span class="p">,</span>
  <span class="na">image</span> <span class="p">=</span> <span class="s">{mypub_imgs_protein_sectors.png}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tesileanu, Tiberiu and Colwell, Lucy J. and Leibler, Stanislas}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pcbi.1004091}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLOS Computational Biology}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e1004091}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Protein Sectors: Statistical Coupling Analysis versus Conservation}}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{protalign}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.plos.org/10.1371/journal.pcbi.1004091}</span><span class="p">,</span>
  <span class="na">pdf</span> <span class="p">=</span> <span class="s">{https://journals.plos.org/ploscompbiol/article/file?id=10.1371/journal.pcbi.1004091&amp;type=printable}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2015}</span>
<span class="p">}</span></code></pre></figure>
    </div>
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-3 d-flex justify-content-center">
<!-- check for dark version of the image --><!-- automatically generate dark version name by adding '-dark' before ext --><div class="photo"><img class="photo-image" src="/public/img/pub/mypub_imgs_mpra.png"></div>
</div>

  <!-- Entry bib key -->
  <div id="Melnikov2012" class="col-sm-9">
    <!-- Title -->
    <div class="title"><a href="https://www.nature.com/articles/nbt.2137" target="_blank" rel="noopener noreferrer">Systematic dissection and optimization of inducible enhancers in human cells using a massively parallel reporter assay</a></div>
    <!-- Author -->
    <div class="author">Melnikov, A., Murugan, A., Zhang, X., <em>Tesileanu, T.</em>, Wang, L.<i> et al.</i>
</div>

    <!-- Journal/Book title and date -->
    <div class="periodical">
      <em>Nature Biotechnology</em> Feb 2012
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a>
    </div>

    <!-- Hidden abstract block -->
    <div class="abstract hidden">
      <p>Learning to read and write the transcriptional regulatory code is of central importance to progress in genetic analysis and engineering. Here we describe a massively parallel reporter assay (MPRA) that facilitates the systematic dissection of transcriptional regulatory elements. In MPRA, microarray-synthesized DNA regulatory elements and unique sequence tags are cloned into plasmids to generate a library of reporter constructs. These constructs are transfected into cells and tag expression is assayed by high-throughput sequencing. We apply MPRA to compare ≥27,000 variants of two inducible enhancers in human cells: a synthetic cAMP-regulated enhancer and the virus-inducible interferon-β enhancer. We first show that the resulting data define accurate maps of functional transcription factor binding sites in both enhancers at single-nucleotide resolution. We then use the data to train quantitative sequence-activity models (QSAMs) of the two enhancers. We show that QSAMs from two cellular states can be combined to design enhancer variants that optimize potentially conflicting objectives, such as maximizing induced activity while minimizing basal activity.</p>
    </div>
<!-- Hidden bibtex block -->
    <div class="bibtex hidden">
      <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Melnikov2012</span><span class="p">,</span>
  <span class="na">image</span> <span class="p">=</span> <span class="s">{mypub_imgs_mpra.png}</span><span class="p">,</span>
  <span class="na">bibtex_show</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">selected</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Melnikov, Alexandre and Murugan, Anand and Zhang, Xiaolan and Tesileanu, Tiberiu and Wang, Li and Rogov, Peter and Feizi, Soheil and Gnirke, Andreas and Callan, Curtis G. and Kinney, Justin B. and Kellis, Manolis and Lander, Eric S. and Mikkelsen, Tarjei S.}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/nbt.2137}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Nature Biotechnology}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{271--277}</span><span class="p">,</span>
  <span class="na">pmid</span> <span class="p">=</span> <span class="s">{22371084}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Nat Biotech}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{transcription}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{Systematic dissection and optimization of inducible enhancers in human cells using a massively parallel reporter assay}}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.nature.com/articles/nbt.2137}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{30}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2012}</span>
<span class="p">}</span></code></pre></figure>
    </div>
  </div>
</div>
</li>
</ol>
          </div>

</article>

</div>

        <!-- Social -->
        <div class="social">
          <div class="contact-icons">
          <a href="mailto:%74%74%65%73%69%6C%65%61%6E%75@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fas fa-envelope"></i></a>
            <a href="https://scholar.google.com/citations?user=8--imZAAAAAJ" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a>
            <a href="https://github.com/ttesileanu" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a>
            <a href="https://www.linkedin.com/in/ttesileanu" title="LinkedIn" target="_blank" rel="noopener noreferrer"><i class="fab fa-linkedin"></i></a>
            <a href="https://twitter.com/ttesileanu" title="Twitter" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a>
            <a href="https://www.quora.com/profile/Tiberiu-Tesileanu" title="Quora" target="_blank" rel="noopener noreferrer"><i class="fab fa-quora"></i></a>
            <a href="https://stackoverflow.com/users/1352250" title="Stackoverflow" target="_blank" rel="noopener noreferrer"><i class="fab fa-stack-overflow"></i></a>
            
          </div>

          <div class="contact-note">
            
          </div>
          
        </div>
      </div>
    </div>

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script src="/public/js/script.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
    <script src="/public/js/common.js"></script>
    
    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

  </body>
</html>
